### ISSUE TITLE
Not working with multiline string

### ISSUE THREAD

**Issue by tnrich** [ZachGawlik/vscode-convert-object-to-jsx#1]
Hey @ZachGawlik I'm excited to use your extension, but for some reason it isn't working. Here is a video of what's happening when I'm trying to convert the following jsx props: 

```js
    toggleOrButton = (
      <EnhancedSwitch
        data-test={type}
        didMount={() => {
          hook && hook(!!(that.state || {})[type]);
        }}
        checked={(that.state || {})[type]}
        label={label ? <span>{label}</span> : type}
        style={{ margin: "0px 30px", marginTop: 4 }}
        onChange={() => {
          hook && hook(!(that.state || {})[type]);
          that.setState({
            [type]: !(that.state || {})[type]
          });
        }}
      />
    );
```

![jsx](https://user-images.githubusercontent.com/2730609/53747563-64c22500-3e58-11e9-90ab-d6c06fb12aae.gif)

It looks like the string is coming into your extension correctly:
![image](https://user-images.githubusercontent.com/2730609/53747763-d5694180-3e58-11e9-9aae-b410e08ec6ed.png)

But the `getProps` function isn't breaking it up into props correctly: 

![image](https://user-images.githubusercontent.com/2730609/53748004-4ad51200-3e59-11e9-8ba0-e1f161c4d7b5.png)

All the props are on a single line of the array!

Let me know if I can you need more info from me or if you're able to reproduce with the snippet I sent you. Thanks!

**Comment by ZachGawlik** [comment]
Hey @tnrich , I really appreciate your detailed explanation and demo of the issue!

The issue comes from the selection not starting at the beginning of the line. The extension is a little simple in how it works, relying on a consistent level of indentation in the selection to determine where to split. I have some time this weekend to look into improving this, as it's not the best user experience and pretty baffling for the user. For now, if you start your selection from the start of the line, you should see it behave more as you'd expect. 

This also pointed out that the extension also needs to add handling hyphenated prop/key names! If you've already been poking around in the codebase this should be a relatively simple fix, otherwise I can also look into it this weekend. 

Thanks again!

**Comment by tnrich** [comment]
Hey @zachgawlik I tried to get it to work earlier but wasn't able to get it. If you could fix it I would definitely appreciate it! It's just the package I'd been looking for!

**Comment by ZachGawlik** [comment]
@tnrich Fixed in v0.4.0!

![vscode-selection-fix](https://user-images.githubusercontent.com/5553498/54079117-b37a1f80-429b-11e9-9bed-3c7efa6f5fb2.gif)

**Comment by tnrich** [comment]
Awesome thanks so much Zach! Much appreciated!
